boards/arm/stm32l562e_dk/stm32l562e_dk.dts

```diff
@@ -35,6 +35,10 @@
 	pinctrl-0 = <&usart1_tx_pa9 &usart1_rx_pa10>;
 };
 
+&usart3 {
+	pinctrl-0 = <&usart3_tx_pc10 &usart3_rx_pc11>;
```

Comment:
Is there a reason to dissociate pinctrl-0 defintion from `status = "okay" `?
Why not moving `pinctrl-0` to file **stm32l562e_dk_common.dtsi**  ?